src/Fixer/Operator/NoUselessConcatOperatorFixer.php

```diff
@@ -336,4 +348,27 @@ private function containsLinebreak(Tokens $tokens, int $startIndex, int $endInde
 
         return false;
     }
+
+    /**
+     * @param array{
```

Comment:
the type should be single line as otherwise not fully fixable

Reply:
> […] not fully fixable

What do you mean? IMO keeping it multiline is better for readiability.

Reply:
Actually, since this type is repeated several times in this class, I'd suggest creating a custom type using `@phpstan-type`

Reply:
Fixed dd9ac1e